addon/components/search/filters.js

```diff
@@ -21,50 +18,16 @@ export default class SearchFiltersComponent extends Component {
     },
   ];
 
-  @tracked changeset;
-
-  constructor(owner, args) {
-    super(owner, args);
-    this.changeset = new Changeset(
-      {
-        realestateIdentification: {
-          EGRID: null,
-          subDistrict: null,
-        },
-        createDate: {
-          dateFrom: null,
-          dateTo: null,
-        },
-        modifyDate: {
-          dateFrom: null,
-          dateTo: null,
-        },
-        ...this.args?.model,
-      },
-      lookupValidator(this.args.validations),
-      this.args.validations
-    );
-  }
-
-  get isSubmitBtnDisabled() {
+  get isSubmitDisabled() {
     return !this.hasChanges;
   }
 
   get hasChanges() {
-    return !!this.changeset.changes.find(({ value }) => value !== "");
+    return !!this.args.changeset.changes.find(({ value }) => value !== "");
```

Comment:
Nitpick for better readability:
```suggestion
    return Boolean(this.args.changeset.changes.find(({ value }) => value !== ""));
```

Reply:
done